test/ethereum/network/evmScanTxProcessing.cases.ts

```diff
@@ -0,0 +1,273 @@
+import { EdgeTransaction } from 'edge-core-js'
+
+import { TransactionProcessingContext } from '../../../src/ethereum/networkAdapters/EvmScanAdapter'
+import { allTokensMapFixture } from './allTokensMapFixture'
+import { currencyInfoFixture } from './currencyInfoFixture'
```

Comment:
Can we use the actual Ethereum info, instead of copy-pasting everything? Alternatively, can we trim down the fixtures cases to just the tokens / currency info we care about?